web/src/lib/components/shared-components/drag-and-drop-upload-overlay.svelte

```diff
@@ -1,16 +1,20 @@
 <script lang="ts">
 	import { fade } from 'svelte/transition';
 	import ImmichLogo from './immich-logo.svelte';
-
+	import { page } from '$app/stores';
 	export let dropHandler: (event: DragEvent) => void;
 
 	let dragStartTarget: EventTarget | null = null;
+
+	const handleDragEnter = (e: DragEvent) => {
+		if ($page.route.id?.includes('(user)')) {
```

Comment:
Probably better to do this check in the root `+layout.svelte` to keep the component as generic as possible and allow it to unmount on pages that don't need it. Something like this:

```svelte
{#if $page.route.id?.startsWith('/(user)')}
	<UploadCover {dropHandler} />
{/if}
```